### DISCUSSION TITLE
Message vertical align issues

### DISCUSSION THREAD

**Discussion by karl-police** [Chatterino/chatterino2#3520] [Ideas]
### Checklist

- [X] I'm reporting a problem with Chatterino
- [X] I've verified that I'm running **the most** recent nightly build or stable release
- [X] I've looked for my problem on the [wiki](https://wiki.chatterino.com/Help/)
- [X] I've searched the [issues and pull requests](https://github.com/Chatterino/chatterino2/issues?q=) for similar looking reports

### Describe your issue

Messages aren't vertically centered in the "message element" depending on what is inside of it.

Example:

![](https://i.imgur.com/ilg4a9I.png)

Not sure if this is related https://github.com/Chatterino/chatterino2/issues/2642

### OS and Chatterino Version

Chatterino Nightly 2.3.4 (commit 820099821, built on )

**Comment by zneix** [comment]
Question is, do you consider this as something that should be changed or just reporting the way Chatterino shows messages. Should we center the messages vertically? If so, what is the intention behind this?
Another question, what about multiline messages? What would be the desired position of "username elements" and "message content elements".

**Comment by karl-police** [comment]
> Question is, do you consider this as something that should be changed or just reporting the way Chatterino shows messages. Should we center the messages vertically? If so, what is the intention behind this? Another question, what about multiline messages? What would be the desired position of "username elements" and "message content elements".

Twitch originally has all messages vertically centered.

For multiline

![image](https://user-images.githubusercontent.com/12023782/149595140-4008cc35-5dc5-4a35-9de1-a35e2391b6da.png)

The username seems to be a "inline" part of the message, but it's vertically centered within that rectangle.

The intention would be to make it more like original Twitch chat.

I noticed though that it seems to be all fine in Chatterino, unless there's an emote, then it's not centered.

See
![image](https://user-images.githubusercontent.com/12023782/149595485-00379677-a267-4666-b8d7-f6364db7785e.png)

![image](https://user-images.githubusercontent.com/12023782/149595515-0be032f6-6523-48c5-a453-b47648052922.png)
